fastchat/model/model_adapter.py

```diff
@@ -1382,6 +1382,8 @@ def load_model(self, model_path: str, from_pretrained_kwargs: dict):
         return model, tokenizer
 
     def get_default_conv_template(self, model_path: str) -> Conversation:
+        if "mistral-7b-openorca" in model_path.lower():
+            return get_conv_template("mistral-7b-openorca")
```

Comment:
In function `def match(self, model_path: str):`, it should return "True/Flase" instead of `get_conv_template`.
Could you fix it?

Reply:
Got it. I will fix that in a bit.

Reply:
Required changes have been made and pushed.